lib/mirage.ml

```diff
@@ -1525,6 +1570,9 @@ module Project = struct
   let version = Mirage_version.current
   let prelude =
     "open Lwt\n\
+     let die fmt =\n\
+       Format.kfprintf (fun _ -> exit 1) Format.err_formatter\n\
+         (\"*** Setup failure ***@\\n@\\n\" ^^ fmt ^^ \"@.\")\n\
```

Comment:
Maybe. It would be unfortunate if the error occurred before logging was configured though. Then the error wouldn't appear at all.